src/web/components/Onwards/OnwardsLower.tsx

```diff
@@ -17,6 +17,8 @@ export const OnwardsLower = ({ ajaxUrl, hasStoryPackage, tags }: Props) => {
     );
 
     let url;
+    let typeOfOnwards: TypeOfOnwards = 'default-onwards';
+
```

Comment:
In OnwardsLower the type should always be `series` so this default is a bit moot but on the other hand, maybe it's nice to keep the possibility to do something else? 🤷‍♂

Reply:
Yeah, that's fair. I guess it just means if someone *does* add something, messes up the logic and we start passing through `default-onwards` into analytics it gives us visibility.

Reply:
Makes sense. Lets leave as is